README.md

```diff
@@ -77,7 +77,13 @@ Page](https://maxmind.github.io/minfraud-api-php/) under the "API" tab.
 
 ## Usage ##
 
-To use this API, create a new `\MaxMind\MinFraud` object. The constructor
+This library provides access to both the [minFraud2 (Score, Insights and
```

Comment:
"minFraud2" is an internal name. We should just use "minFraud" in public documentation. The same applies below.